### PR TITLE
improved cypress testing to cover most of use cases

### PR DESCRIPTION
**Pull request by rexalex**

### CODE REVIEW

File: cypress/integration/01-slate-zotero-block.js [hunk 1]
```diff
@@ -0,0 +1,540 @@
...
+    // the two items in the first collection
+    cy.intercept(
+      'GET',
+      'https://api.zotero.org/users/6732/collections/24C33/items/?start=0&limit=10&sort=title',
+      {
+        statusCode: 200,
+        headers: {
+          'Total-Results': '14',
+        },
+        body: [
```
Comment (nileshgulia1) [inline cypress/integration/01-slate-zotero-block.js:235]: You can also move this inside `fixtures/` to make it look more concise.

**Comment by avoinea** [comment]
@rexalex Is this ready? As I can see cypress tests are failing.

**Review by avoinea — CHANGES_REQUESTED** [review]
Jenkins is red

File: cypress/integration/01-slate-zotero-block.js (hunk 1 above)
Comment (rexalex) [reply cypress/integration/01-slate-zotero-block.js]: fixed

**Review by nileshgulia1 — APPROVED** [review]

**Comment by nileshgulia1** [comment]
The kitkat tests should pass after this gets in https://github.com/eea/volto-eea-kitkat/tree/develop

**Pull request merged**